src/data/fintype/basic.lean

```diff
@@ -50,6 +50,19 @@ by rw [inter_comm, univ_inter]
   {δ : α → Sort*} (f g : Πi, δ i) : univ.piecewise f g = f :=
 by { ext i, simp [piecewise] }
 
+lemma univ_map_equiv_to_embedding {α : Type*} [fintype α] (e : α ≃ α) :
```

Comment:
This lemma doesn't even need the two types to be the same, right?
The proof could be shortened using `eq_univ_iff_forall`.

Reply:
Statement and proof both improved, thanks!